openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -311,10 +325,6 @@ <h1 translate>Agenda</h1>
       <!-- main content column -->
       <div class="col-xs-6 content"
         style="padding-left: calc({{ items.length === itemsSearched.length ? item.parentCount : 0 }}*25px)">
-        <div class="icon-column">
```

Comment:
if you remove `icon-column` you also can remove the `title-column`.

Reply:
these classes are just for the purpose, that the div's are next to each other

Reply:
Done